### PR TITLE
Shareable message

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/search/get-started-with-search/search-basics/share-message-link.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+You can share a link to a specific message in the Sumo Logic search table, which allows you to quickly and easily share important information. To share a message link to a search, after your query has run, click **Message** tab beneath the search query box.
+
+This link will be available for three years after it is created. 
+
+To share a message link to a search:
+
+1. Run a search you'd like to share.
+2. Once the search results are generated, navigate to the **Message** tab. <br/> <img src={useBaseUrl('/img/search/get-started-search/search-basics/copy-message.png')} alt="copy-message" width="900"/>
+3. Find the message you want to share and right-click on it. Then, select **Copy link to message**. <br/> <img src={useBaseUrl('/img/search/get-started-search/search-basics/select-message.png')} alt="icon" width="800"/>
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-basics/share-message-link.md:23]: Titlecased to match how it appears in the UI

```suggestion
3. Find the message you want to share and right-click on it. Then, select **Copy Link to Message**. <br/> <img src={useBaseUrl('/img/search/get-started-search/search-basics/select-message.png')} alt="icon" width="800"/>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Looks great! Just a minor tweak.

File: docs/search/get-started-with-search/search-basics/share-message-link.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Sumo Logic enables you to share links to specific messages in the search table. This is an extended functionality of the [Share a link to search](/docs/search/get-started-with-search/search-basics/share-link-to-search/) feature.
+
+You can share a link to a specific message in the Sumo Logic search table, which allows you to quickly and easily share important information. To share a message link to a search, after your query has run, click **Message** tab beneath the search query box.
+
+This link will be available for three years after it is created. 
+
+To share a message link to a search:
+
+1. Run a search you'd like to share.
+2. Once the search results are generated, navigate to the **Message** tab. <br/> <img src={useBaseUrl('/img/search/get-started-search/search-basics/copy-message.png')} alt="copy-message" width="900"/>
```
Comment (JV0812) [inline docs/search/get-started-with-search/search-basics/share-message-link.md:22]: **Messages** tab.

Comment (JainM6) [inline docs/search/get-started-with-search/search-basics/share-message-link.md:22]:
```suggestion
2. Once the search results are generated, navigate to the **Messages** tab. <br/> <img src={useBaseUrl('/img/search/get-started-search/search-basics/copy-message.png')} alt="copy-message" width="900"/>
```

File: docs/search/get-started-with-search/search-basics/share-message-link.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::important
+Search results might vary depending on each user's permissions.
+:::
+
+Sumo Logic enables you to share links to specific messages in the search table. This is an extended functionality of the [Share a link to search](/docs/search/get-started-with-search/search-basics/share-link-to-search/) feature.
+
+You can share a link to a specific message in the Sumo Logic search table, which allows you to quickly and easily share important information. To share a message link to a search, after your query has run, click **Message** tab beneath the search query box.
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-basics/share-message-link.md:15]:
```suggestion
You can share a link to a specific message in the Sumo Logic search table, which allows you to quickly and easily share important information. To share a message link to a search, after your query has run, click **Messages** tab beneath the search query box.
```

**Review by JV0812 — APPROVED** [review]

**Pull request merged**